.babelrc

```diff
@@ -0,0 +1,16 @@
+{
+  "env": {
+    "test": {
+      "presets": [
+        [
+          "@babel/preset-env",
+          {
+            "targets": {
+              "node": "current"
+            }
+          }
+        ]
+      ]
+    }
+  }
+}
```

Comment:
What is this file used for?

Reply:
Babel configuration

Reply:
What does this configuration say? I have a `babel.config.js` that says:
```
module.exports = {
  presets: ["@vue/cli-plugin-babel/preset"]
};
```

Reply:
Oh, sorry. I'm not sure, it's just nuxt default